why-jsonschema/index.qmd

```diff
@@ -254,12 +258,35 @@ What decision was made, use the form "We decided on CHOICE because of
 REASONS."
 :::
 
-TBD
+We decided to use `jsonschema` for automating checks on metadata. It is
+a great match for our use case: it offers a single functionality with a
+robust implementation, integrates into our codebase with minimal work,
+and makes it easy to keep Sprout up-to-date with the Data Package
+standard.
+
+Although `frictionless-py` implements the Data Package standard
+directly, it is less mature and less widely used than `jsonschema`. We
+have found some inconsistencies in its verification/validation behaviour
+and feel that we would need to customise it using somewhat brittle and
+inelegant workarounds for it to fit into our workflow. Pydantic would
+have been a fair choice, but it would have involved the most integration
+and customisation work, resulting in far more complicated code to
+achieve the same behaviour as the other options.
 
 ### Consequences
 
 ::: content-hidden
 List some potential consequences of this decision.
 :::
 
-TBD
+-   Any recommendations not included in the schema of the Data Package
+    standard will have to be added (if needed) as custom validators.
+-   We will need to decide how we want to treat blank values, as some of
+    these violate the schema (e.g. empty lists).
+-   If we want to keep checks for different types of metadata separate,
+    we will need to write custom logic for this.
+-   We will have to think about how we want to incorporate the schema of
+    the Data Package standard into Sprout. Although the Data Package
+    standard is hosted as a [project on
+    GitHub](https://github.com/frictionlessdata/datapackage), it is
+    maintained using npm, so we cannot add it directly as a dependency.
```

Comment:
Should these be bullet points for emphasis?

Reply:
Do you mean a numbered list? :)

Reply:
Yeah because I think they are bullet points?